kinder/cmd/kinder/do/do.go

```diff
@@ -120,6 +121,12 @@ func NewCommand() *cobra.Command {
 		"ignore-preflight-errors", constants.KubeadmIgnorePreflightErrors,
 		"list of kubeadm preflight errors to skip",
 	)
+	cmd.Flags().StringVar(
+		&flags.KubeadmConfigVersion,
+		"kubeadm-config-version", constants.DefaultKubeadmConfigVersion,
+		fmt.Sprintf("the kubeadm config version to be used for init, join and upgrade; use one of %s",
```

Comment:
What about adding: "if not set, kubeadm will automatically choose the Kubeadm config version according to the Kubernetes version in use"

Reply:
make sense!